### PR TITLE
CPIB CUSP-Me to EQR120 building array recorder swap

### PR DESCRIPTION
**Pull request by Thomas-Benson**
Replacing CUSP-Me recorders at CWHB with EQR120s
 - EQR120 with serial numbers 46788 - 46795 installed at locations 21 - 2F in numerical order
 - All new recorders have orientation 0 and firmware version 2.3.2

Updated streams from HN-XYZ to HN-12Z

### CODE REVIEW

File: install/recorders.csv [hunk 1]
```diff
@@ -793,6 +793,21 @@ Canterbury Seismic Instruments,EQR120,EQR120,46777,WQAB,27,10,,0,0,2024-05-22T02
 Canterbury Seismic Instruments,EQR120,EQR120,46778,WQAB,28,10,,0,0,2024-05-22T02:22:00Z,9999-01-01T00:00:00Z
 Canterbury Seismic Instruments,EQR120,EQR120,46779,WQAB,29,10,,0,0,2024-05-22T02:22:00Z,9999-01-01T00:00:00Z
 Canterbury Seismic Instruments,EQR120,EQR120,46780,WQAB,2A,10,,0,0,2024-05-22T02:22:00Z,9999-01-01T00:00:00Z
+Canterbury Seismic Instruments,EQR120,EQR120,46781,CPIB,21,0,,0,2.2,2024-07-20T07:57:00Z,9999-01-01T00:00:00Z
+Canterbury Seismic Instruments,EQR120,EQR120,46782,CPIB,22,0,,0,2.2,2024-07-20T07:57:00Z,9999-01-01T00:00:00Z
+Canterbury Seismic Instruments,EQR120,EQR120,46783,CPIB,23,90,,0,-0.5,2024-07-20T07:57:00Z,9999-01-01T00:00:00Z
```
Comment (mnaguit) [inline install/recorders.csv:798]: all EQRs are on 0 degree azimuth?

**Review by mnaguit — COMMENTED** [review]
Thank you @Thomas-Benson , prior to approval, I'll wait until the streams and azimuths are updated.

**Review by CBurton90 — DISMISSED** [review]

**Comment by Thomas-Benson** [comment]
Sorry for the re-review requests, had to resolve a merge conflict with the other Cusp to EQR upgrade PR

**Review by CBurton90 — APPROVED** [review]

**Comment by mnaguit** [comment]
From John's comment in the main ticket, all EQRs are in 0 azimuth, unlike the old CUSP system?

**Review by CBurton90 — DISMISSED** [review]

**Review by mnaguit — APPROVED** [review]

**Review by CBurton90 — APPROVED** [review]

**Pull request merged**